app/javascript/components/Bookmarks.jsx

```diff
@@ -86,7 +87,17 @@ const Bookmark = ({ bookmark, editable, setEditable }) => {
   return (
     <div className={'card-list-item is-' + bookmark.bookmark_class_name}>
       <div className="card-list-item__inner">
-        <div className="card-list-item__label">{bookmark.modelNameI18n}</div>
+        {bookmark.modelName === 'Talk' ? (
+          <div className="card-list-item__user">
+            <UserIcon
+              user={bookmark.user}
+              blockClassSuffix='card-list-item'
+            />
+          </div>
+        ) : (
+          <div className="card-list-item__label">{bookmark.modelNameI18n}</div>
+        )
+        }
```

Comment:
行が違うのにインデントレベルが違うと不自然かなと思うので下記の方がいいかもです。

```suggestion
        )}
```

Reply:
こちら修正いたしました！